test/registries.conf

```
@@ -1,10 +1,20 @@
 # Note that changing the order here may break tests.
-[registries.search]
-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
+unqualified-search-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
 
 [registries.insecure]
 registries = []
 
 #blocked (docker only)
 [registries.block]
 registries = []
+
+# In Nov. 2020, Docker rate-limits image pulling.  To avoid hitting these
+# limits while tesing, use the provided google cache.  Integration tests
+# make use of the copy of this file under `./test/`, however system tests
+# use the globally installed file.
+# Ref: https://cloud.google.com/container-registry/docs/pulling-cached-images
+[[registry]]
+location="docker.io"
```

Assuming we don't want to pull from `docker.io` at all, we should do the following:
```suggestion
prefix="docker.io"
location="mirror.gcr.io"
```

If we configure `gcr` as mirror, we can still fall back to using docker.io.

---

Oh...interesting idea, I like it.